src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_Hierarchy.cs

```diff
@@ -293,7 +307,7 @@ async Task AddReferencingSymbols(ReferencedSymbol reference, bool isDirect)
                 {
                     var collector = new StreamingProgressCollector();
                     await FindReferencesAsync(
-                        symbolAndProjectId, solution, collector, documents,
+                        symbol, solution, collector, documents,
                         FindReferencesSearchOptions.Default, cancellationToken).ConfigureAwait(false);
                     return collector.GetReferencedSymbols();
                 }
```

Comment:
the _Legacy.cs file below is gone.  All the ops we previous made 'legacy' are now fully supported.  the only downside is they produce IEnumreables instead of IMmutableArrays.